### DISCUSSION TITLE
[BUG] There is a bug in the nuclear templates

### DISCUSSION THREAD

**Discussion by EmptyZeroRain** [projectdiscovery/nuclei-templates#10872] [Q&A] [answered]
The specific link can be found at https://github.com/projectdiscovery/nuclei/issues/5679

**Comment by EmptyZeroRain** [comment]
nuclei

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.3.4

		projectdiscovery.io

[WRN] Found 2 templates with runtime error (use -validate flag for further examination)

**Comment by EmptyZeroRain** [comment]
[ERR] Error occurred parsing template  nuclei-templates/http/technologies/wordpress/plugins/wp-mail-logging.yaml: could not compile request: could not parse payloads: the helpers/wordpress/plugins/wp-mail-logging.txt file for payload last_version does not exist or does not contain enough elements

**Comment by ritikchaddha** [comment, marked as answer]
This has been fixed https://github.com/projectdiscovery/nuclei-templates/issues/10908
